plugins/utils/makeFeedHtml.js

```diff
@@ -0,0 +1,53 @@
+const { imageWrapperClass } = require('gatsby-remark-images/constants')
+const unified = require('unified')
+const { convertHastToHtml } = require('./convertHast.js')
+const { select, selectAll } = require('hast-util-select')
+
+const rootToAbsolute = siteUrl => tree => {
+  selectAll('a', tree).forEach(node => {
+    if (node.properties.href.startsWith('/')) {
+      node.properties.href = siteUrl + node.properties.href
+    }
+  })
+  selectAll('img', tree).forEach(node => {
+    if (node.properties.src.startsWith('/')) {
+      node.properties.src = siteUrl + node.properties.src
+    }
+  })
+  return tree
+}
+
+const unwrapImages = () => tree => {
+  selectAll(`.${imageWrapperClass}`, tree).forEach(node => {
+    // Set the fallback image as the wrapper's only child, and then
+    // give that image the wrapper's original style.
+    const fallbackImg = select('img', node)
+    node.children = [
+      {
+        ...fallbackImg,
+        properties: {
+          ...fallbackImg.properties,
+          style: 'max-width: 100%; margin: auto;'
```

Comment:
qq-  does it makes/do we respect our custom image sizing logic?

Reply:
This just changes the wrapper's internal image to have `max-width: 100%` so larger images don't extend offscreen on narrower displays and `margin: auto` so smaller ones are centered. 
The original `max-width` on the wrapper that the existing plugin operates on is untouched, as well as any other properties on the wrapper.